openassessment/xblock/job_sample_grader/code_grader.py

```diff
@@ -28,6 +28,16 @@ class TestGrader:
     __TMP_DATA_DIR__ = "/tmp/"
 
     def grade(self, response, add_staff_cases=False):
+        """
+        Grade submission based on correctness. Place submission code in a file named
```

Comment:
Not necessarily on correctness. If the problem is designed-based, no correctness checks are needed.